klippy/extras/bed_mesh.py

```diff
@@ -415,6 +444,80 @@ def print_probed_positions(self, print_func):
             print_func(msg)
         else:
             print_func("bed_mesh: bed has not been probed")
+
+
+    def tilt_probe_finalize(self, offsets, positions):
+        x_offset = offsets[0]
+        y_offset = offsets[1]
+        z_offset = offsets[2]
+        self.gcode.respond_info \
+                ("offsets: %f %f %f" % (x_offset,y_offset,z_offset) );
+
+        params=self.bedmesh.z_mesh.probe_params
+
+        if self.relative_reference_index is not None:
+            # zero out probe z offset and
+            # set offset relative to reference index
+            z_offset = positions[self.relative_reference_index][2]
+
+        if self.probed_z_table_backup is None:
+            self.probed_z_table_backup = copy.deepcopy(self.probed_z_table)
+
+        t_probed_z_table=copy.deepcopy(self.probed_z_table_backup)
+        self.bedmesh.z_mesh.build_mesh(t_probed_z_table)
```

Comment:
I don't think its necessary to rebuild the mesh here.  The results should be identical the the current mesh state.